utils/comment.go

```diff
@@ -225,52 +262,30 @@ func generateReviewComment(commentType ReviewCommentType, location formats.Locat
 
 }
 
-func generateApplicabilityReviewContent(issue formats.Evidence, relatedCve formats.CveRow, relatedVulnerability formats.VulnerabilityOrViolationRow, writer outputwriter.OutputWriter) string {
-	remediation := ""
-	if relatedVulnerability.JfrogResearchInformation != nil {
-		remediation = relatedVulnerability.JfrogResearchInformation.Remediation
+func generateApplicabilityReviewContent(issue issues.ApplicableEvidences, writer outputwriter.OutputWriter) string {
```

Comment:
this function is used only once, there is no need to have this function we can simply call
 outputwriter.GenerateReviewCommentContent(outputwriter.ApplicableCveReviewContent(issue, writer), writer)
 where generateApplicabilityReviewContent is called